### PR TITLE
Consumer group details view.

### PR DESCRIPTION
**Pull request by soffest**

### CODE REVIEW

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                  {...{clusterName, ...consumer}}
+                />
+              ))}
+          </tbody>
+        </table>
+      </div>
+    </div>
+  );
+};
+
```
Comment (workshur) [inline kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:67]: Please check the indentation

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+              <th>Current offset</th>
+              <th>End offset</th>
+            </tr>
+          </thead>
+          <tbody>
+            {items
+              .map((consumer, index) => (
+                <ListItem
+                  key={`consumers-list-item-key-${index}`}
+                  {...{clusterName, ...consumer}}
```
Comment (workshur) [inline kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:58]:
```suggestion
                  clusterName={clusterName}
                  {...consumer}
```

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/ListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Consumer } from 'redux/interfaces/consumerGroup';
+import { NavLink } from 'react-router-dom';
+import { ClusterName } from '../../../redux/interfaces/cluster';
```
Comment (workshur) [inline kafka-ui-react-app/src/components/ConsumerGroups/Details/ListItem.tsx:4]:
```suggestion
import { ClusterName } from 'redux/interfaces/cluster';
```

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx (hunk 1 above)
Comment (soffest) [reply kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:68]: Checked indentation and I don't see any issues here, it also corresponds to indentation in similar existing files in the project.

**Pull request merged**